install.sh

```diff
@@ -117,10 +117,10 @@ execute() {
 
 execute_sudo() {
   local -a args=("$@")
-  if [[ -n "${SUDO_ASKPASS-}" ]]; then
-    args=("-A" "${args[@]}")
-  fi
   if have_sudo_access; then
+    if [[ -n "${SUDO_ASKPASS-}" ]]; then
+      args=("-A" "${args[@]}")
+    fi
     ohai "/usr/bin/sudo" "${args[@]}"
     execute "/usr/bin/sudo" "${args[@]}"
   else
```

Comment:
```ruby
execute_sudo() {
  if have_sudo_access; then
    if [[ -n "${SUDO_ASKPASS-}" ]]; then
      ohai "/usr/bin/sudo" -A "$@"
      execute "/usr/bin/sudo" -A "$@"
    else
      ohai "/usr/bin/sudo" "$@"
      execute "/usr/bin/sudo" "$@"
    fi
  else
```
Stylistic suggestion. I haven't tested this code.

Thanks for the quick fix, Hielke!

Reply:
I see it was this PR was merged while I was typing up this comment! Feel free to open a PR with this stylistic change if you prefer it, but also feel free to leave it as is.

Reply:
Well, we could just get rid of the whole -A in args together. Same `have_sudo_access`. 

But this already got merged, let's save this for another time.